eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -179,13 +179,11 @@ stages:
                           ReleaseSha: $(Build.SourceVersion)
                           RepoId: Azure/azure-sdk-for-python
                           WorkingDirectory: $(System.DefaultWorkingDirectory)
```

Comment:
FYI @danieljurek this is just a small part of the clean-up we need to do for the docs processing once you refactor it for the daily packages as well.